server/lobbyconnection.py

```diff
@@ -941,6 +930,22 @@ def command_modvault(self, message):
             else:
                 raise ValueError('invalid type argument')
 
+    @asyncio.coroutine
+    async def command_ice_servers(self, message):
+        if self.player:
+
+            ttl = 24*3600
+            ice_servers = self.coturn_generator.fetch_token(self.player.id, ttl)
```

Comment:
Do we wanna calculate the longterm-credential token anyways even if no static auth secret is defined and twilio is going to be used?